embuilder.py

```diff
@@ -235,115 +178,17 @@ def is_flag(arg):
         print('Skipping building of native-optimizer; EMSCRIPTEN_NATIVE_OPTIMIZER set in .emscripten config.')
       else:
         tasks += ['native_optimizer']
+    tasks += ['struct_info']
```

Comment:
why is this needed?

Reply:
`generated_struct_info.json` is no longer automatically built due to the build system change, so we make this task run explicitly when building all libraries.

Reply:
please add a comment explaining that, as otherwise it's very surprising to see it singled out for special treatment. something like "is necessary for building anything else". but that's not the full explanation, since isn't libc in a similar situation?

Reply:
It might make more sense for this to be listed in system targets.